concordium-consensus/src/Concordium/GlobalState/AccountMap.hs

```diff
@@ -0,0 +1,186 @@
+{-# LANGUAGE DataKinds #-}
```

Comment:
Minor thing: Why not place the pure and the persistent ones (account maps) in separate modules, such that the pure and persistent block states doesn't get exposed of functions that they should not use.

Reply:
Because they are so trivial that I thought having a separate Basic.AccountMap (that would have to import the global one) would just be a lot of noise and would not gain value.